client/src/main/java/io/split/client/dtos/SplitHttpResponse.java

```diff
@@ -0,0 +1,10 @@
+package io.split.client.dtos;
+
+/**
+ * A structure for returning http call results information
+ */
+public class SplitHttpResponse {
+    public Integer statusCode;
+    public String statusMessage;
+    public String body;
```

Comment:
can we add response headers as well? Also, how about making this class immutable? so that no manipulation can take place and it becomes inherently thread-safe

Reply:
will do